boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts

```diff
@@ -115,6 +115,17 @@
 	pinctrl-names = "default";
 	status = "okay";
 	clock-frequency = <I2C_BITRATE_FAST>;
+
+	stmpe811@41 {
+		compatible = "st,stmpe811";
+		status = "okay";
+		reg = <0x41>;
+		int-gpios = <&gpioa 15 GPIO_ACTIVE_LOW>;
+
+		kscan_input: kscan-input {
+			compatible = "zephyr,kscan-input";
+		};
```

Comment:
This can be dropped now, LVGL has a native input driver and these have been removed from all touchscreen, if your application needs it it should be in the application overlay.

Reply:
Thanks! This is removed